modules/tracking/samples/lasot_benchmark.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+import cv2 as cv
+import glob
+import argparse
+
+def init_tracker(tracker_name):
+    if tracker_name == "Boosting":
+        tracker = cv.TrackerBoosting_create()
+        frames_before_reinit = 500
+    elif tracker_name == "MIL":
+        tracker = cv.TrackerMIL_create()
+        frames_before_reinit = 1000
+    elif tracker_name == "KCF":
+        tracker = cv.TrackerKCF_create()
+        frames_before_reinit = 110
+    elif tracker_name == "MedianFlow":
+        tracker = cv.TrackerMedianFlow_create()
+        frames_before_reinit = 20
+    elif tracker_name == "GOTURN":
+        tracker = cv.TrackerGOTURN_create()
+        frames_before_reinit = 20
+    elif tracker_name == "MOSSE":
+        tracker = cv.TrackerMOSSE_create()
+        frames_before_reinit = 20
+    elif tracker_name == "CSRT":
+        tracker = cv.TrackerCSRT_create()
+        frames_before_reinit = 250
+    return [tracker, frames_before_reinit]
+
+def main():
+    parser = argparse.ArgumentParser(description="Run LaSOT-based benchmark for visual object trackers")
+    parser.add_argument("--path_to_dataset", type=str,
+                        default="Test part of LaSOT dataset", help="Full path to LaSOT folder")
+    args = parser.parse_args()
+
+    list_of_videos = [None for _ in range(280)]
+    file_with_video_names = open(args.path_to_dataset + "/testing_set.txt", "r")
+
+    for i in range(len(list_of_videos)):
+        list_of_videos[i] = file_with_video_names.readline()
+
+    trackers = {'Boosting': 500, 'MIL': 1000, 'KCF': 110,
+                'MedianFlow': 20, 'GOTURN': 20, 'MOSSE': 20, 'CSRT': 250}
+
+    #For every tracker
+    for tracker_id in range(len(list(trackers.keys()))):
+
+        tracker_name = list(trackers.keys())[tracker_id]
+        print(tracker_name)
+
+        #For every video
+        for video_name in list_of_videos:
+
+            tracker, frames_before_reinit = init_tracker(tracker_name)
+            init_once = False
+
+            print("Video: " + str(video_name))
+
+            video_name = video_name.replace("\n", "")
+            gt_file = open(
+                args.path_to_dataset + video_name + "/groundtruth.txt", "r")
+            gt_bb = gt_file.readline().replace("\n", "").split(",")
+            init_bb = gt_bb
+
+            for _ in range(len(init_bb)):
+                init_bb[_] = float(init_bb[_])
```

Comment:
Please, use i or another name to iterator. _ is used if iter is not used in the loop

Reply:
In this case better: `init_bb = [float(b) for b in init_bb]`

Reply:
OK.